pkg/operator/encryption/deployer/staticpod.go

```diff
@@ -51,14 +56,23 @@ func NewStaticPodDeployer(
 	return &StaticPodDeployer{
 		podClient:                podClient.Pods(targetNamespace),
 		secretClient:             secretClient.Secrets(targetNamespace),
-		targetNamespaceInformers: informers.InformersFor(targetNamespace),
+		operatorClient:           operatorClient,
+		targetNamespaceInformers: namespaceInformers.InformersFor(targetNamespace),
 	}, nil
 }
 
 // DeployedEncryptionConfigSecret returns the deployed encryption config and whether all
 // instances of the operand have acknowledged it.
 func (d *StaticPodDeployer) DeployedEncryptionConfigSecret() (secret *corev1.Secret, converged bool, err error) {
-	revision, err := getAPIServerRevisionOfAllInstances(d.podClient)
+	_, status, _, err := d.operatorClient.GetStaticPodOperatorState()
+	if err != nil {
+		return nil, false, err
+	}
+	if status == nil || len(status.NodeStatuses) == 0 {
```

Comment:
what does it mean? no nodes?

Reply:
Maybe status was wiped or cluster is installing.